third-party/linshare/src/test/java/org/apache/james/linshare/LinshareTest.java

```diff
@@ -58,4 +65,14 @@ void linshareShouldStart() {
         .then()
             .statusCode(HttpStatus.SC_OK);
     }
+
+    @Test
+    void linshareShouldShouldHaveATechnicalAccountConfigured() {
+        List<TechnicalAccountDTO> technicalAccounts = linshareExtension.getAllTechnicalAccounts(LinshareFixture.ADMIN_ACCOUNT);
+
+        assertThat(technicalAccounts).anySatisfy(account -> SoftAssertions.assertSoftly(softly -> {
```

Comment:
Just curious, can't we also assert that there is only one technical account?

Reply:
When start docker linshare already had a TechnicalAccount and it didn't like this one we created.